<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="LogInTest"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="0.037">
  <properties />
            <testcase name="loginTest" time="0.037" classname="com.w2a.testcases.LoginTest">
                <failure type="java.lang.IllegalArgumentException"
                  message="Cannot find elements when the selector is null"
              >
<![CDATA[java.lang.IllegalArgumentException: Cannot find elements when the selector is null
  org.openqa.selenium.By$ByCssSelector.<init>(By.java:422)
  org.openqa.selenium.By.cssSelector(By.java:116)
  com.w2a.base.Page.click(Page.java:158)
  com.w2a.pages.HomePage.goToLogin(HomePage.java:22)
  com.w2a.testcases.LoginTest.loginTest(LoginTest.java:18)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.base/java.lang.reflect.Method.invoke(Method.java:567)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
  org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
  org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
  org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
  org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
  org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
  java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
  org.testng.TestRunner.privateRun(TestRunner.java:770)
  org.testng.TestRunner.run(TestRunner.java:591)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
  org.testng.SuiteRunner.run(SuiteRunner.java:304)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
  org.testng.TestNG.runSuites(TestNG.java:1032)
  org.testng.TestNG.run(TestNG.java:1000)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
